docs/getting-started/installation-and-upgrade/installation-references/tls-settings.md

```diff
@@ -23,3 +23,75 @@ The default TLS configuration only accepts TLS 1.2 and secure TLS cipher suites.
+
+This setting defaults to `strict` on new installs of Rancher (starting with v2.9.0), but is set as `system-store` for users installing/upgrading from a prior version.
+
+### Preparing for the Setting Change
+
+Each cluster contains a condition in the status field called `AgentTlsStrictCheck`. If this value is set to true, this indicates the agents for the cluster are ready to operate using the `strict` mode. You can manually inpsect each cluster to see if they are ready using the Rancher UI, or a Kubectl command like the following:
+
+```bash
+## the below command skips ouputs $CLUSTER_NAME,$STATUS for all non-local clusters
+kubectl get cluster.management.cattle.io -o jsonpath='{range .items[?(@.metadata.name!="local")]}{.metadata.name},{.status.conditions[?(@.type=="AgentTlsStrictCheck")].status}{"\n"}{end}'
+```
+
+### Changing the Setting
+
+You can change the setting using the Rancher UI or the `agentTLSMode` [helm chart option](./helm-chart-options.md).
+
+:::note
+
+If the value is set through the helm chart, it must be changed through the helm chart.
+
+:::
+
+:::warning
+
+Depending on your cert setup, additional action (such as uploading the Certificate Authority which signed your certs) may be required. Review the [installation guide](../install-upgrade-on-a-kubernetes-cluster/install-upgrade-on-a-kubernetes-cluster.md#3-choose-your-ssl-configuration) before changing the setting to see if any additional requirements apply to your setup.
+
+:::
+
+To change the option through the UI by navating to the "Global Settings" page, and looking for the `agent-tls-mode` setting near the bottom of the page. When changing the setting through the UI, Rancher will check to ensure that all downstream clusters have the status check set to `true`, in order to avoid outages from a certificate mismatch.
+
+
+#### Overriding the Setting Validation Checks
+
+In some cases, you may want to override the check ensuring all agent's can accept the new TLS configuration. To override this check, you must update the setting using the following steps:
+
+:::warning
+
+Rancher checks the status of all clusters to prevent downstream cluster outages. Overriding this check is not recommended, and should be done with great caution.
+
+:::
+
+1. As an admin, generate a kubeconfig for the local cluster. In the below examples, this was saved to the `local_kubeconfig.yaml` file.
+2. Using the following command, retrieve the current setting, and save it to the `setting.yaml` file:
+```bash
+kubectl get setting agent-tls-mode -o yaml --kubeconfig=local_kubeconfig.yaml >> setting.yaml
+```
+3. Update the yaml file, replacing `value` with `strict`. Adding the `cattle.io/force: true` annotation will override the above check, and should only be done with great care:
```

Comment:
```suggestion
3. Update the setting file, replacing `value` with `strict`. Adding the `cattle.io/force: true` annotation will override the cluster condition check, and should only be done with great care:
```

Reply:
We don't actually check the value of the annotation, only that it's present - the key could have any value.

Reply:
I think that level of nuance would be too much for these docs (and honestly we should be check for the value set to true).

Reply:
Right, I think that is something that we should do. But it's separate from this PR. More specifically, the point of this docs is to give people something that they can run to bypass the check. 

What I've provided here does work and is accurate, it's just doesn't explain the full range of options. But I think that's ok - I'm not aware of a reason why a user would want to use anything other than what was provided. At most, I thinnk we would want to include a warning that states that values than `"true"` also work, and you should use caution with this annotation. WDYT?

Reply:
I just hope people won't try to "turn off" the overriding annotation not by removing it but by setting the value to "false", which will not do what they expect.

Reply:
Yeah I can add the warning which explains that doesn't work if you would like. That way the solution is clear and precise, but there's a warning about that potential common path.